docs/latest/dev/CONTRIBUTING.md

```diff
@@ -44,8 +44,8 @@ to the following guidelines for all code, APIs, and documentation:
 * PRs are expected to have 100% test coverage for added code. This can be verified with a coverage
   build. If your PR cannot have 100% coverage for some reason please clearly explain why when you
   open it.
-* Any PR that changes user-facing behavior **must** have associated documentation in [docs](docs) as
```

Comment:
We could keep the link to `docs` by using the absolute URL `https://github.com/envoyproxy/gateway/tree/main/docs`